libmscore/rendermidi.cpp

```diff
@@ -731,6 +735,478 @@ bool Score::isSubdivided(ChordRest* chord, int swingUnit)
             return false;
       }
 
+void renderTremolo(Chord *chord, QList<NoteEventList> & ell){
```

Comment:
Indentation of these new functions is completely off. There is no comment before the function. Idem for renderArpeggio()